git/githistory/ref_updater.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/hex"
 	"fmt"
 	"strings"
+	"io"
+	"bytes"
```

Comment:
I suspect our Go linter will ask for these to be ordered alphabetically, so that's a small change we'll need before the CI suite would pass.